apps/s_client.c

```diff
@@ -1336,6 +1367,22 @@ int s_client_main(int argc, char **argv)
             max_version = DTLS1_2_VERSION;
             socket_type = SOCK_DGRAM;
             isdtls = 1;
+# ifndef OPENSS_NO_QUIC
+            isquic = 0;
+# endif
+#endif
+            break;
+        case OPT_QUIC:
+#ifndef OPENSSL_NO_QUIC
+            meth = OSSL_QUIC_client_method();
+            /* TODO(QUIC): What should these values be for QUIC? */
+            min_version = TLS1_3_VERSION;
```

Comment:
A good question. What happens to APIs and values for the next version of QUIC?

Reply:
Initial updates for QUIC version handling were made in #20650. How that applies to min/max version handling was extensively debated in #20790 and subsequently in this week's OTC. The result is #20830. I've updated this PR to be in line with #20830.